imports/ui/components/parties/party_list.html

```diff
@@ -1,13 +1,6 @@
 <template name='partyList'>
-  <div class='party-header'>
-    {{#if currentUser}}
-      <a href='/events/new'><button type="button" class="btn btn-default btn-md party-header__button">
-        <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Create
-      </button></a>
-    {{/if}}
-  </div>
-  <div class='clear'></div>
-  <div class='parties'>
+  <h4 class='news__title'>Events</h4>
+  <div class='news-item'>
```

Comment:
I guess the news title and item CSS works here, but it does look a bit awkward having them in the party list, right? This is fine for now but I wonder if there's some other name we could use.

Reply:
I agree that it does look awkward, but I don't think there is a need to define another class with the same attributes.